inference-engine/tests/functional/plugin/gna/Import_export_tests/import_export_memory_layer.cpp

```diff
@@ -66,8 +75,10 @@ class ImportMemoryTest : public testing::WithParamInterface<exportImportNetworkP
         auto importedNetwork = core->ImportNetwork(inputStream, targetDevice, configuration);
         std::vector<std::string> queryToState;
         InferenceEngine::InferRequest importInfer = importedNetwork.CreateInferRequest();
-        for (const auto &query_state : importInfer.QueryState()) {
+        for (auto &query_state : importInfer.QueryState()) {
             queryToState.push_back(query_state.GetName());
+            if (withReset.second)
+                query_state.Reset();
```

Comment:
How does this additional parameter affect test checks? Maybe check memory state after reset at least?